emulator/blockchain.go

```diff
@@ -1641,3 +1643,8 @@ func (b *Blockchain) GetTransactionResultsByBlockID(blockID flowgo.Identifier) (
 	}
 	return results, nil
 }
+
+func (b *Blockchain) SetTimestamp(timestamp int64) {
+	b.timestamp = timestamp
+	b.pendingBlock.SetTimestamp(b.timestamp)
+}
```

Comment:
Maybe we can do this something like `SetTimeDelta`. Most use cases will be about `moving time` ( offer expires after 1 day, valid until 1 year etc )

Reply:
That's a valid point :+1: I just opened the PR to get some brainstorming on what's the best solution to tackle this problem.

Reply:
@bluesign Do you have any API blueprint for your suggestion? To use as a point of reference.

Reply:
I think this is good for this PR, we can later advance on this. ( maybe add little mocking capabilities etc )

Reply:
As a developer, would you prefer the `SetTimestamp` to have effect only for the next block? Or it should be persistent until the developer decides to move the back in the present? With something like `Time.now()`